go.mod

```diff
@@ -2,7 +2,7 @@ module github.com/gballet/go-verkle
 
 go 1.18
 
-require github.com/crate-crypto/go-ipa v0.0.0-20230601170251-1830d0757c80
+require github.com/crate-crypto/go-ipa v0.0.0-20230626131944-6a9b06cf26df
```

Comment:
As mentioned in the PR description, this is targeting a tentative go-ipa version that doesn't have a PR yet.
It exists mostly to assist go-verkle in this PR. We can figure out when it can make sense to make it official after the tests.